src/commands/update.ts

```diff
@@ -0,0 +1,68 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import { bin_name, config } from '..'
+import { log } from '../log'
+
+import {
+  setupFirefoxSource,
+  shouldSetupFirefoxSource,
+} from './download/firefox'
+import {
+  addAddonsToMozBuild,
+  downloadAddon,
+  generateAddonMozBuild,
+  getAddons,
+  initializeAddon,
+  resolveAddonDownloadUrl,
+  unpackAddon,
+} from './download/addon'
+import { getLatestFF } from '../utils'
+import { rmSync } from 'node:fs'
+import { ENGINE_DIR } from '../constants'
+
+export const update = async (): Promise<void> => {
+  const version = await getLatestFF(config.version.product)
```

Comment:
Do you ever update `gluon.json` with the new version?

Reply:
That's true, mb

Reply:
Updated